packages/next-dashboard/src/components/layout/Header.js

```diff
@@ -25,19 +24,30 @@ const Header = ({ toggleSidebarActive, sidebarActive, children }: Props) => (
           <div className="header-content-container">
             <div className="grid grid-x3-medium grid-x4-large align-items-center">
               <div className="cell dashboard-header-cell dashboard-header-cell_sidebar">
-                <button
-                  type="button"
-                  className="dashboard-header-sidebar-button"
-                  onClick={toggleSidebarActive}
-                >
-                  <span
-                    className={[
-                      'dashboard-sidebar-button-icon',
-                      'fa',
-                      sidebarActive ? 'fa-times' : 'fa-bars',
-                    ].join(' ')}
-                  />
-                </button>
+                <div className="hamburger-menu">
+                  <label>
+                    <input
+                      type="checkbox"
+                      onChange={toggleSidebarActive}
+                      checked={sidebarActive}
+                      defaultChecked={sidebarActive}
+                    />
+                    <svg
+                      viewBox="0 0 100 100"
+                      xmlns="http://www.w3.org/2000/svg"
+                    >
+                      <path
+                        className="line--1"
+                        d="M0 40h62c13 0 6 28-4 18L35 35"
+                      />
+                      <path className="line--2" d="M0 50h70" />
+                      <path
+                        className="line--3"
+                        d="M0 60h62c13 0 6-28-4-18L35 65"
+                      />
+                    </svg>
+                  </label>
+                </div>
```

Comment:
It would probably be a good idea to break this out into its own `<HamburgerMenu />` component, I think. (The details of how the hamburger is constructed, with SVG paths etc, isn't really relevant to the Header component as a whole, so it's nice to separate those bits out)